src/commands/config/enable-karma-tracking-config.ts

```diff
@@ -0,0 +1,27 @@
+import type { Firestore } from '@google-cloud/firestore';
+import type { GuildTextBasedChannel } from 'discord.js';
+
+import { getChannelFirestoreReference } from '../../utilities/firestore-helper.js';
+
+/** Enables or disables karma tracking on a channel in Firebase */
+export async function enableKarmaTrackingConfig(
+  firestore: Firestore,
+  channel: GuildTextBasedChannel,
+  value: boolean | null
+): Promise<string> {
+  // TODO figure out how to fetch this correctly
+  if (!channel) {
+    throw new Error();
+  }
+
+  // Set value in database
+  const docRef = getChannelFirestoreReference(firestore, channel);
+  await docRef.set({ karmaTracking: value }, { merge: true });
+
+  // Enable / disable feature
```

Comment:
floating comment? Doesn't seem to be referring to anything specific

Reply:
Ah yes. Originally I was going to refactor the karma tracking to create & destroy listeners based on the config being set instead of checking the db for every reaction, but decided against that (in this PR at least).

Will remove comment and if you think its a good idea to change this logic int he future we can make an issue and do it later.